src/twitch/clip-downloader.ts

```diff
@@ -59,5 +58,6 @@ export class ClipDownloader extends EventEmitter {
         }
 
         await Promise.all(promises);
+        return Promise.resolve(clip);
```

Comment:
Also return `clip` directly.

Reply:
Clip is returned line 61

Reply:
You don't need to wrap in `Promise.resolve`, you can:
```suggestion
        return clip;
```